metadata-db-app/src/cmr/metadata_db/migrations/074_add_permission_notification.clj

```diff
@@ -0,0 +1,18 @@
+(ns cmr.metadata-db.migrations.074-add-permission-notification
+  "Adds fingerprint to variables table"
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up to version 74."
+  []
+  (println "cmr.metadata-db.migrations.074-add-permission-notification up...")
+  (h/sql "alter table METADATA_DB.cmr_sub_notifications add permission_check_time TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL")
+  (h/sql "alter table METADATA_DB.cmr_sub_notifications add permission_check_failed INTEGER DEFAULT 0 NOT NULL"))
```

Comment:
I don't think we need two fields to track since when the ACL check starts failing. We could use just one field called e.g. `permission_failed_since` to save the time the ACL check start failing. 
If ACL check fails, 
  - permission_failed_since is not present, we update permission_failed_since to the current time
  - permission_failed_since is present, we don't need to update the notification
 If ACL check succeeds, 
  - permission_failed_since is not present, we don't need to update the notification
  - permission_failed_since is present, we update permission_failed_since to NULL